projects/github.com/hadolint/hadolint/package.yml

```diff
@@ -0,0 +1,30 @@
+distributable:
+  url: https://github.com/hadolint/hadolint/archive/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: hadolint/hadolint
+  strip: /^v/
+
+build:
+  dependencies:
+    haskell.org: ~9.2
```

Comment:
This version isn't packaged yet, is that easy to do?

Reply:
I'll give it a go.

Reply:
https://github.com/teaxyz/pantry/actions/runs/5348749798